web/app/themes/justicejobs/single-agency-image.php

```diff
@@ -8,76 +8,76 @@
 <?php get_header(); ?>
 
 <section class="hero hero--agency">
-  <div class="hero__img-block">
-    <div
-      class="hero__img hero__img--desktop"
-      style="background-image: url('<?php the_field( 'agency_hero_desktop_image' ); ?>');"
-    ></div>
-    <div
-      class="hero__img hero__img--mobile"
-      style="background-image: url('<?php the_field( 'agency_hero_mobile_image' ); ?>');"
-    ></div>
-    <div class="hero__img_description">
-      <?php the_field( 'agency_hero_image_description' ); ?>
+    <div class="hero__img-block">
+        <div
+            class="hero__img hero__img--desktop"
+            style="background-image: url('<?php the_field('agency_hero_desktop_image'); ?>');"
+        ></div>
+        <div
+            class="hero__img hero__img--mobile"
+            style="background-image: url('<?php the_field('agency_hero_mobile_image'); ?>');"
+        ></div>
+        <div class="hero__img_description">
+            <span class="text-highlight"><?php the_field('agency_hero_image_description'); ?></span>
+        </div>
+        <div class="hero__text-wrap">
+            <svg class="hero__arrow hero__arrow--top" width="37" height="24">
+                <use xlink:href="#icon-arrow--decor"></use>
+            </svg>
+            <ul class="breadcrumbs text-highlight">
+                <li><a href="<?php echo get_bloginfo('url'); ?>">Home</a></li>
+                <li>Agency</li>
+            </ul>
+            <h1 class="heading--lg"><?php the_title(); ?></h1>
+            <svg class="hero__arrow hero__arrow--bottom" width="37" height="24">
+                <use xlink:href="#icon-arrow--decor"></use>
+            </svg>
+        </div>
+        <div class="hero__badge">
+            <img
+                src="<?php the_field('agency_logo_black'); ?>"
+                width="164"
+                height="77"
+                alt="HM Courts & Tribunals Service"
```

Comment:
I realise this is probably only listing as changed due to formatting but while i spotted it can you change it so the alt tag for the logo is 'the title; not a hard coded string. This is also the case on single-agency.php. Its been there for a while but since we changing this file anyway. Thought it be good to fix a small bug.

Reply:
Yes of course.